core/src/main/java/com/google/cloud/sql/core/DefaultAccessTokenSupplier.java

```diff
@@ -135,6 +135,28 @@ public Optional<AccessToken> get() throws IOException {
     }
   }
 
+  private static boolean isAccessTokenEmpty(GoogleCredentials credentials) {
+    return credentials.getAccessToken() == null
+        || "".equals(credentials.getAccessToken().getTokenValue());
+  }
+
+  private void refreshIfNearExpiration(GoogleCredentials credentials) throws IOException {
```

Comment:
nit: Since we also refresh for reasons other than expiry maybe a better name is refreshIfRequired or refreshIfNecessary or something similar.

Reply:
Fixed.